src/components/ChatBox.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState, useRef, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { io } from 'socket.io-client';
+import {
+  getActiveMeeting,
+  addMessageActiveMeeting,
+} from '../redux/slices/meetingSlice';
+
+const socket = io.connect('http://localhost:3333');
+
+export default function ChatBox() {
+  const dispatch = useDispatch();
+  const [newMessage, setNewMessage] = useState('');
+  const meeting = useSelector((state) => state.meetings.meeting);
+  const auth = useSelector((state) => state.auth.user);
+  const today = new Date();
+  const dayOfMonth = today.getUTCDate();
+  const monthToday = today.getMonth();
+  const yearToday = today.getUTCFullYear();
+
+  useEffect(() => {
+    const asyncStart = async () => {
+      const token = localStorage.getItem('token');
+      const disMeeting = await dispatch(getActiveMeeting(token));
+      socket.emit('joinRoom', disMeeting.payload.id);
+    };
+    asyncStart();
+    setTimeout(() => {
+      const scrollAnchor = document.getElementById('scroll-here');
+      scrollAnchor?.scrollIntoView();
+    }, 500);
+  }, []);
+
+  useEffect(() => {
+    // REMOVE .OFF WHEN DEPLOYING OR ELSE WILL NEVER SEND MSG
+    socket.off('recieve-message').on('recieve-message', (d) => {
+      const asyncEvent = async () => {
+        const token = localStorage.getItem('token');
+        setTimeout(() => {
+          dispatch(getActiveMeeting(token));
+        }, 500);
+        setTimeout(() => {
+          const scrollAnchor = document.getElementById('scroll-here');
+          scrollAnchor.scrollIntoView();
+        }, 1000);
+      };
+      asyncEvent();
+    });
+  }, [socket]);
+
+  const onMessageSubmit = async (e) => {
+    e.preventDefault();
+    if (newMessage === '') return;
+    else {
+      const token = localStorage.getItem('token');
+      dispatch(addMessageActiveMeeting({ token, newMessage }));
+      socket.emit('message-event', meeting.id);
+      setNewMessage('');
+    }
+  };
+
+  const handleEnterClick = (e) => {
+    if (e.key === 'Enter' && !e.shiftKey) {
+      onMessageSubmit(e);
+    }
+  };
+  // checks if user is logge in
+  if (!auth.id) {
+    return (
+      <h1>
+        Please{' '}
+        <Link to="/login" className="text-blue-400">
```

Comment:
Change text color to match what other similar links look like. Also it may be better to put the classname on a p tag that is wrapped in the link